components/form/Form.tsx

```diff
@@ -50,6 +48,7 @@ const InternalForm: React.ForwardRefRenderFunction<unknown, FormProps> = (props,
     scrollToFirstError,
     requiredMark,
     onFinishFailed,
+    name,
```

Comment:
Thanks for your contribution. 

But we need to pass `name` to `rc-filed-form`.

The `restFormProps` will drop `name` if you use destruction.

Reply:
add test?